docs/use-cases/navigate.md

```diff
@@ -1,8 +1,54 @@
 ---
-title: "Teleoperate and navigate rovers"
+title: "Teleoperate and navigate with rovers"
 linkTitle: "Teleoperate and navigate with rovers"
-weight: 60
+weight: 50
 type: "docs"
-layout: "empty"
-canonical: "tutorials/services/navigate-with-rover-base/"
+tags: ["navigation", "motion", "services", "base", "base remote control"]
+no_list: true
+description: "Use Viam to teleoperate and navigate rover bases."
+image: "/services/icons/navigation.svg"
+imageAlt: "Navigation"
+images: ["/services/icons/navigation.svg"]
 ---
+
+If you have a rover base, you can teleoperate and navigate it with Viam.
+Remotely control your base after configuring your machine on the app's **Control** tab, and set up autonomous navigation with the [navigation service](/mobility/navigation/).
```

Comment:
```suggestion
```
I *think* this sentence could actually be removed since it's sort of just a reiteration and expansion of the first sentence, but you're going to cover all these points below too, so don't need to get into the details here.

You might consider adding a sentence about why you'd want to do this with Viam, such as that all these tools are integrated seamlessly, or something.